### PR TITLE
 Add Construction Tag

### PR DESCRIPTION
**Pull request by Bentleysb**
### Description:

Adds a tag interface for the construction tag. This tag uses values from the bridge, highway, landuse, and railway tags. 

### Potential Impact:

Adds a new tag class. 

### Unit Test Approach:

None

### Test Results:

None

------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: src/main/java/org/openstreetmap/atlas/tags/ConstructionTag.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.openstreetmap.atlas.tags.annotations.Tag;
+import org.openstreetmap.atlas.tags.annotations.TagKey;
+
+/**
+ * OSM Construction tag. Inherits values from Bridge, Highway, LandUse and Railway tags.
+ *
+ * @author bbreithaupt
+ */
+@Tag(with = { BuildingTag.class, HighwayTag.class, LandUseTag.class,
```
Comment (sayas01) [inline src/main/java/org/openstreetmap/atlas/tags/ConstructionTag.java:11]: 🔥

File: src/main/java/org/openstreetmap/atlas/tags/ConstructionTag.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        RailwayTag.class }, taginfo = "https://taginfo.openstreetmap.org/keys/construction", osm = "https://wiki.openstreetmap.org/wiki/Key:construction")
+public interface ConstructionTag
+{
+    @TagKey
+    String KEY = "construction";
```
Comment (sayas01) [inline src/main/java/org/openstreetmap/atlas/tags/ConstructionTag.java:17]: You might get "Constants should not be defined in interfaces (squid : S1214)" sonar warning. You may want to suppress it.

**Review by sayas01 — APPROVED** [review]
LGTM! 🎉

File: src/main/java/org/openstreetmap/atlas/tags/ConstructionTag.java (hunk 2 above)
Comment (Bentleysb) [reply src/main/java/org/openstreetmap/atlas/tags/ConstructionTag.java:17]: Good point. Well remembered!

**Review by nachtm — APPROVED** [review]
LGTM

**Pull request closed without merging**